share/completions/git.fish

```diff
@@ -374,6 +378,12 @@ function __fish_git_reflog
     command git reflog --no-decorate 2>/dev/null | string replace -r '[0-9a-f]* (.+@\{[0-9]+\}): (.*)$' '$1\t$2'
 end
 
+function __fish_git_help_all_concepts
+    git help -g | string match -e -r '^   \w+' | while read -l concept desc
+        complete -f -c git -n '__fish_git_using_command help' -a $concept -d (string trim $desc)
```

Comment:
This is all topsy-turvy, and generates the candidates when the completion file is loaded.

What it should do instead is

```fish
complete -f -c git -n '__fish_git_using_command help' -a '(git help -g [...])'
```

with the description separated from the concept with a tab character (`\t`).